frontend/src/components/commons/Notepad.jsx

```diff
@@ -0,0 +1,96 @@
+import React, { Component } from "react";
+import LOCALIZE from "../../text_resources";
+import TextareaAutosize from "react-textarea-autosize";
+import { BROWSER_STRING, IE_STRING, VALID_BROWSERS } from "../../Status";
+
+let COLS = "";
```

Comment:
Since COLS gets adjusted within the class, it can't be a const, and therefore should not be all caps. 
It would be better to make this a state variable called `columnWidth` or something similar.